css/index.css

```diff
@@ -172,4 +172,91 @@ header .logo {
 footer {
     text-align: center;
     margin: 40px 0 20px;
-}
\ No newline at end of file
+}
+
+/* Tablet formatting */
+
+@media (max-width: 920px) {
+  .container {
+    max-width: 880px;
+		width: calc(100% - 40px);
+    margin: 0px 20px;
+  }
+
+	header .logo {
+    margin: 28px 0 0 40px;
+	}
+
+	.cta {
+    justify-content: space-around;
+	}
+
+	.cta .cta-text {
+		margin: 0;
+		width: 40%;
+	}
+
+	.main-content {
+	}
+
+	.main-content .middle-img {
+		display: flex;
+		width: 100%;
+	}
+}
+
+/* Mobile formatting */
+
+@media (max-width: 750px) {
+	header {
+    flex-wrap: wrap;
+    height: 100%;
+  }
+
+  header nav {
+    width: 100%;
+    flex-direction: column;
+    align-items: center;
+  }
+
+	header nav a {
+		height: 30px;
+		border-style: solid;
+		border-top-width: 1px;
+		border-color: gray;
+		align-self: stretch;
+		text-align: center;
+	}
+
```

Comment:
Weird inconsistencies with the white space above and below this point. You use Atom right? I use Prettier with ESLint and it helps a ton with stuff like this.